### ISSUE TITLE
#18 Consider Alternative Loss Functions for Stability

### ISSUE THREAD

**Issue by veeradi34** [veeradi34/Hello-foss-Video-summarization#22]
Binary cross-entropy loss (BCELoss) can sometimes lead to training instability in GANs. To achieve more stable training, consider using alternative losses, such as the Wasserstein GAN (WGAN) loss, which minimizes the Earth-Mover (Wasserstein) distance and can reduce training oscillations.
